recipes/artery-font-format/all/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile, CMake, tools
+
+
+class ArteryFontFormatConan(ConanFile):
+    name = "artery-font-format"
+    license = "MIT"
+    homepage = "https://github.com/Chlumsky/artery-font-format"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Artery Atlas Font format library"
+    topics = ("conan", "artery", "font", "atlas")
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+        tools.tools_files.rename("artery-font", "include/artery-font")
```

Comment:
```suggestion
```

Reply:
Done in 86a61a8c2d2839b0aa54c78d4282630d28b9f90c.